files/ipset_sync

```diff
@@ -119,11 +119,11 @@ function ipset_insync_common() {
   local pfx=$2
 
 
-  # compare configured and runtime config
+  # compare configured and runtime config, ignoring hashsize, initval and bucketsize parameters
   tf=$(mktemp)
-  diff \
-    <(get_ipset_dump ${id} | grep ^${pfx} | sed 's/hashsize [0-9]\+ //g') \
-    <(construct_ipset_dump ${id} | grep ^${pfx} | sed 's/hashsize [0-9]\+ //g') \
+  diff -w \
+    <(get_ipset_dump ${id} | grep ^${pfx} | sed -e 's/hashsize [0-9]\+ //g' -e 's/initval 0x[0-9a-f]\+//g' -e 's/bucketsize [0-9]\+ //g') \
```

Comment:
Extra space is not always removed after the parameter value (e.g. `s/hashsize [0-9]\+ //g` vs `s/initval 0x[0-9a-f]\+//g`).

This command could be replaced by `sed -e 's/ hashsize [0-9]\+//g' -e 's/ initval 0x[0-9a-f]\+//g' -e 's/ bucketsize [0-9]\+//g' -e 's/[[:space:]]*$//')

Reply:
@sid3windr would it be possible to change it accordingly?

Reply:
@maxadamo I rebased the PR and it could be merged now. Did you test the suggestion from @oliparcol ?

Reply:
Any update on this perhaps @sid3windr?

Reply:
@bastelfreak yes, that change works.

Reply:
@bastelfreak , @JoshuaSmeda  the suggestion from @oliparcol is correct, but 
1. change on line 126 is not necessary
2. we're trying to move this commands inside the function. 
I'm not sure if we create a new PR or we try to change this one.